boussole/parser.py

```diff
@@ -94,7 +94,7 @@ def filter_rules(self, path):
         * Starts with http:// or https:// (this for external load only)
         * Ends with ".css" (they are not intended to be compiled)
         """
-        return not(
+        return not (
```

Comment:
It does not look needed at all, revert this line change.

Reply:
I don't mind, but this line causes flake8 to fail:
```log
.venv/bin/flake8 --show-source boussole
boussole/parser.py:97:19: E275 missing whitespace after keyword
        return not(
                  ^
make: *** [Makefile:96: flake] Error 1
```

Reply:
Hmm that's strange, i may have missed a breakward flake release, ok you can forget this, i will see myself when releasing.